superbench/benchmarks/micro_benchmarks/cublas_function/cublas_benchmark.h

```diff
@@ -0,0 +1,340 @@
+/**
```

Comment:
For c++ source file, we should add header:



<!--StartFragment-->// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.<!--EndFragment-->